book/content/open_research/03/openhardware.md

```diff
@@ -26,46 +26,63 @@ Here are some files that you should consider sharing when publishing your open s
 You are not required to post them all, but the more you share the more the community benefits.
 There is a lot of crossover here with files to include in open source software projects.
 
-- Overview / Introduction: Your open source hardware project should include a general description of the hardware's identity and purpose, written as much as possible for a general audience.
+#### Overview and introduction
+Your open source hardware project should include a general description of the hardware's identity and purpose, written as much as possible for a general audience.
 That is, explain what the project is and what it is for before you get into the technical details.
 A good photo or rendering can help a lot here.
```

Comment:
```suggestion

```

Reply:
We can add this requirement in an issue rather than the content

Reply:
I think the original author (I think this was @mjg211 who can verify?) was suggesting that a photo can be helpful as an accompaniment to the hardware, i.e. in the documentation, rather than a photo is helpful in the Turing Way book?